### DISCUSSION TITLE
construct community models without abundance

### DISCUSSION THREAD

**Discussion by Ql-cy** [micom-dev/micom#197] [Q&A]
Hi, Thanks for developing this useful tools! I have a question that I have a taxonomy file and feature table, if I set all value in the feature table to 1(seems setting the relative abundance to 1 in all samples), does this mean that I only retrived the genome-scale draft models from the database and combined them into a community? Beacuse I dont care about the abundace. I am not sure that make sense~Thanks!

**Comment by cdiener** [comment]
Yes exactly. Either setting the abundances to same value or not having any abundance column as well will yield a community model where all taxa have the same relative abundance of 1/n.

**Reply by winatony** [reply]
Is this **explicitly** stated in the documentation? Because it should be, I have been hunting for the answer to this myself. Thank you for answering this.

**Reply by cdiener** [reply]
It is, but only for the [single model API](https://micom-dev.github.io/micom/community.html) where it says:

> As you can notice we have gained a new column called abundance. This column quantifies the relative quantity of each individual in the community. Since we did not specify this in the original taxonomy micom assumes that all individuals are present in the same quantity.

You are right that this should be mentioned in the workflows. I will add it in the next doc update or you are also very much invited to send in a PR.

**Reply by winatony** [reply]
Along these lines, if you build a single community using the `Community` function, can you also identify interactions? Currently the documentation for `interactions` seems to suggest that it requires the generation of a database of communities (sample info, etc.).
